lib/ansible/modules/messaging/rabbitmq/rabbitmq_user.py

```diff
@@ -249,7 +248,7 @@ def main():
         read_priv=dict(default='^$'),
         force=dict(default='no', type='bool'),
         state=dict(default='present', choices=['present', 'absent']),
-        node=dict(default='rabbit'),
+        node=dict(default=None),
```

Comment:
Omit `default` if there isn't one. But instead, please add `type`:
```suggestion
        node=dict(type='str'),
```